.github/workflows/release.yml

```
@@ -0,0 +1,157 @@
+        
+        if [ -z "$PREVIOUS_TAG" ]; then
+          # First release, get all commits
+          COMMITS=$(git log --oneline --no-merges)
+        else
+          # Get commits since last release
+          COMMITS=$(git log --oneline --no-merges ${PREVIOUS_TAG}..HEAD)
+        fi
+        
+        # Get PR information
+        PRS=$(gh pr list --state merged --search "merged:>${PREVIOUS_TAG:-1970-01-01}" --json number,title,url --jq '.[] | "- #\(.number): \(.title) (\(.url))"' 2>/dev/null || echo "")
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)